### PR TITLE
Add persian language and some txt and xml files for dictionary

### PR DESCRIPTION
**Pull request by mhdzli**
Hello,

The persian language was added before and works greate. in this PR I add  some txt and xml files for dictionary though the "./gradlew :languages:persian:pack:makeDictionary" does not build the dictionary from inputs and my xml backup from AnySoftKeyboard (Using android SDK and linux terminal and then android studio, both don't do the job). Is there something that I missed? I can add my result log of makeDictionary result. 
Anyway thank you for all the work on this project!

Regards,
Mohammad

### CODE REVIEW

**Comment by menny** [comment]
thank you. I'll look into that.

**Comment by mhdzli** [comment]
> thank you. I'll look into that.

Sorry to bother you again. Did you find a chance to look into this issue?
I converted my txt file into NextWord and UserWord of anysoft backups and add them to my own keyboard. But it's much better if I can add them to the dictionary of apk file so they will be installed with Persian layout. Thanks a lot.

**Comment by menny** [comment]
@mhdzli please rebase your branch and force-push.
Also, see my comment at https://github.com/AnySoftKeyboard/LanguagePack/pull/274#discussion_r293816326

File: languages/persian/pack/src/main/res/xml/persian_autotext.xml [hunk 1]
```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (mhdzli) [reply languages/persian/pack/src/main/res/xml/persian_autotext.xml]: I missed this file before but now I'll edit it to fit the persian layout.

**Comment by mhdzli** [comment]
> @mhdzli please rebase your branch and force-push.
> Also, see my comment at [#274 (comment)](https://github.com/AnySoftKeyboard/LanguagePack/pull/274#discussion_r293816326)

I rebased my repo and added a new layout for smaller screens

**Pull request closed without merging**